python/cookiecutter-django.md

```diff
@@ -0,0 +1,538 @@
+- **use_mailhog**: This option asks us if we want to add [Mailhog](https://github.com/mailhog/MailHog) as an email viewer during the development process. We recommend that option, so, you should enter: `y`.
+- **use_heroku**: This option asks us if we want to add some configuration files to deploy the project to *Heroku*. At Rootstrap we usually deploy to it, so, you should enter: `y`.
+- **keep_local_envs_in_vcs**: This option asks us if we want to add the environment configurations to the Version Control System. Also, it's useful to set Heroku or docker variables. AAt Rootstrap we manage that information differently, so, you should enter: `n`.
+
+<details>
+  <summary>View full bash output. <i>(Click &#x25B6; to display the file)</i></summary>
+
+```bash
+project_name [My Awesome Project]: Cookiecutter Starter
+project_slug [cookiecutter_starter]:
+description [Behold My Awesome Project!]: This proyect is an example of how configure a Django proyect since beginning
+author_name [Daniel Roy Greenfeld]: Rootstrap, Inc.
+domain_name [example.com]: example.com
+email [rootstrap@example.com]: info@example.com
+version [0.1.0]:
+Select open_source_license:
+1 - MIT
+2 - BSD
+3 - GPLv3
+4 - Apache Software License 2.0
+5 - Not open source
+Choose from 1, 2, 3, 4, 5 [1]: 1
+timezone [UTC]:
+windows [n]:
+use_pycharm [n]:
+use_docker [n]:
+Select postgresql_version:
+1 - 12.3
+2 - 11.8
+3 - 10.8
+4 - 9.6
+5 - 9.5
+Choose from 1, 2, 3, 4, 5 [1]: 1
+Select js_task_runner:
+1 - None
+2 - Gulp
+Choose from 1, 2 [1]:
+Select cloud_provider:
+1 - AWS
+2 - GCP
+3 - None
+Choose from 1, 2, 3 [1]: 1
+Select mail_service:
+1 - Mailgun
+2 - Amazon SES
+3 - Mailjet
+4 - Mandrill
+5 - Postmark
+6 - Sendgrid
+7 - SendinBlue
+8 - SparkPost
+9 - Other SMTP
+Choose from 1, 2, 3, 4, 5, 6, 7, 8, 9 [1]: 6
+use_async [n]:
+use_drf [n]: y
+custom_bootstrap_compilation [n]: n
+use_compressor [n]:
+use_celery [n]:
+use_mailhog [n]: y
+use_sentry [n]:
+use_whitenoise [n]:
+use_heroku [n]: y
+Select ci_tool:
+1 - None
+2 - Travis
+3 - Gitlab
+Choose from 1, 2, 3 [1]:
+keep_local_envs_in_vcs [y]: n
+debug [n]:
+```
+
+</details>
+
+## Install pipenv
+The recommendation from Rootstrap is that you use `pipenv` to manage the development environment dependencies.
+
+To install the package, just run `$ pip install pipenv`.
+
+## Install dependencies
+To install the dev dependencies, run `$ pipenv install -r requirements/local.txt --dev`.
+
+NOTE:
+- Remember, when you install a new dependency, check if it will be needed at production or not. In the case that it won't be needed, add the flag `--dev` at the end of the installation command.
+- At pipenv to run a command, you must write before `pipenv run`, for example, the command that runs the server should be `$ pipenv run python manage.py runserver`.
```

Comment:
Also you can add that you can execute pipenv shell in the folder to "enter to the environment" and there you don't need to execute `pipenv run` before ` python manage.py runserver`.

Reply:
Sure! I didn't know that trick. I'll add it